src/gui/pages_menu/KM_GUIMenuCampaign.pas

```diff
@@ -201,18 +201,14 @@ procedure TKMMenuCampaign.StartClick(Sender: TObject);
 
 procedure TKMMenuCampaign.AnimNodes(aTickCount: Cardinal);
 begin
-  if fCampaign <> nil then
+  if fCampaign = nil then Exit;
+  if Image_CampaignSubNode[fAnimNodeIndex].Visible then Exit;

```

Comment:
Did you test it with MAX_CAMP_NODES? (64) Think about it, the array Image_CampaignSubNode goes from 0 to 63. When fAnimNodeIndex=63 you make the final node visible (Image_CampaignSubNode[63].Visible := true). This is fine, but then we increment fAnimNodeIndex, so now it is 64. Next time AnimNodes nodes is called it checks "if Image_CampaignSubNode[64].Visible then Exit;". But Image_CampaignSubNode[64] is out of bounds so it will crash.